app/build.gradle

```diff
@@ -79,7 +79,7 @@ jooq {
           driver = 'org.postgresql.Driver'
           url = 'jdbc:postgresql://localhost:5432/postgres'
           user = 'postgres'
-          password = ''
+          password = 'testpassword'
```

Comment:
What's the relationship between this file and `hikariJdbc.properties`. Can this file reference properties from the latter?